.changeset/wild-items-beam.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fluidframework/telemetry-utils": major
+---
+
+Adding support to log a flat object, which will be JSON.stringified.
```

Comment:
```suggestion
Loggers now support logging a flat object, which will be JSON.stringified.
```

Reply:
THis would also be where to describe how to deal with the breaking changes, if applicable. Use your judgement.